src/main/java/com/progzesp/stalking/service/impl/AnswerServiceImpl.java

```diff
@@ -106,15 +108,60 @@ public Pair<Integer, ModifyAnswerEto> modifyAnswer(Long id, ModifyAnswerEto answ
         if (foundEntity.isPresent()) {
             AnswerEntity answerEntity = foundEntity.get();
 
+            UserEntity userEntity = userRepo.getByUsername(user.getName());
 
-            if(!Objects.equals(answerEntity.getGame().getGameMasterId(), userRepo.getByUsername(user.getName()).getId())) {
+            if(!Objects.equals(answerEntity.getGame().getGameMasterId(), userEntity.getId())) {
                 return Pair.of(403, new ModifyAnswerEto());
             }
 
             answerEntity.setChecked(answerEto.isChecked());
             answerEntity.setApproved(answerEto.isApproved());
             answerEntity.setScore(answerEto.getScore());
 
+            if(answerEntity.getGame().getAnswerEntityList() != null) {
+                if (answerEntity.getGame().getEndCondition().equals(EndCondition.SCORE)) {
+                    Integer score = 0;
+                    for (AnswerEntity answerEntityIterate : answerEntity.getGame().getAnswerEntityList()) {
+                        if (answerEntityIterate.isChecked() && answerEntityIterate.isApproved()
+                                && Objects.equals(answerEntityIterate.getUser().getTeamId(), answerEntity.getUser().getTeamId())) {
+                            score += answerEntityIterate.getScore();
+                        }
+                    }
+                    if (score >= answerEntity.getScore()) {
```

Comment:
To nie wydaje się poprawne. Czy nie powinno być przypadkiem porównania score >= game.getEndScore() 

Nie sprawdzałem czy takie pole istnieje